src/pkg/utils/utils.go

```diff
@@ -285,9 +283,13 @@ func GetContainerNamePrefixForImage(image string) (string, error) {
 	return basename, nil
 }
 
-func GetDefaultImageForDistro(distro, release string) string {
+func GetDefaultContainerNamePrefix() string {
+	return containerNamePrefixDefault
```

Comment:
This doesn't seem to address the problem that the hint shown by `create` is influenced by the configuration file.

I think the main complication is that the configuration file lets us specify an image. Earlier we would just pick the host and release from `/etc/os-release`, map that to a fixed image, and go. Now, that mapping can be overridden by the configuration file.

This means that `toolbox create` without any parameters might still use a custom image, and lead to a custom container name.